apps/api/.env.example

```
@@ -33,4 +33,9 @@ DO_SPACE_SECRET_KEY=
 DO_SPACE_REGION=
 DO_SPACE_BUCKET=
 DO_SPACE_ENDPOINT=
-DO_SPACE_CDN_ENDPOINT=
\ No newline at end of file
+DO_SPACE_CDN_ENDPOINT=
+
+#Stripe Configurations
+STRIPE_SECRET_KEY=
+STRIPE_PUBLISHABLE_KEY=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The STRIPE_PUBLISHABLE_KEY key should go before the STRIPE_SECRET_KEY key

(UnorderedKey)